research/ag_news/sparse_tensor_exchange/run_hp_sweep.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+###############################################
+# Usage:
+#
+#  ./research/ag_news/dynamic_layer_exchange/run_hp_sweep.sh \
```

Comment:
I think you want to update the various example paths in this file to `sparse_tensor_exchange` instead of `dynamic_layer_exchange`